app/templates/briefs/_brief_response_data.html

```diff
@@ -0,0 +1,29 @@
+{% import "toolkit/summary-table.html" as summary %}
+{% for section in content.summary(brief_response) %}
+    {% if not (section.id == 'your-nice-to-have-skills-and-experience' and not section.get_question('niceToHaveRequirements').boolean_list_questions) %}
```

Comment:
Exactly - if there are no questions without this we get just the heading displayed.